eng/common/TestResources/build-test-resource-config.yml

```
@@ -47,10 +47,14 @@ steps:
 
       if ($subConfigFilesRaw) {
         $subConfigFiles = $subConfigFilesRaw | ConvertFrom-Json -AsHashtable
```

```suggestion
        $subConfigFiles = @($subConfigFilesRaw | ConvertFrom-Json -AsHashtable)
```
Would that also fix this issue just be ensuring we always have a list?

---

The list would contain an empty string and you'd hit the error of trying to `Get-Content` an empty string. 

```
C:\Users\djurek> $x = @'
>>   ""
>> '@
C:\Users\djurek> $y = @($x | ConvertFrom-Json -AsHashtable)
C:\Users\djurek> $y

C:\Users\djurek> foreach ($item in $y) { Get-Content $item }
Get-Content: Cannot bind argument to parameter 'Path' because it is an empty string.
```

---

I guess with either fix we probably should filter out or skip empty files.